code/modules/cargo/blackmarket/blackmarket_items/consumables.dm

```diff
@@ -0,0 +1,63 @@
+/datum/blackmarket_item/consumable
+	category = "Consumables"
+
+/datum/blackmarket_item/consumable/clown_tears
+	name = "Bowl of Clowns Tears"
```

Comment:
the reagent (and soup) has an apostrophe in "clown's" (at the place where I just put it)